src/SAWScript/Crucible/LLVM/Builtins.hs

```diff
@@ -1350,8 +1350,8 @@ withCfg ::
   IO a
 withCfg context name k =
   do let function_id = L.Symbol name
-     case Map.lookup function_id (Crucible.cfgMap (ccLLVMModuleTrans context)) of
-       Just (_, Crucible.AnyCFG cfg) -> k cfg
+     Crucible.getTranslatedCFG (ccLLVMModuleTrans context) function_id >>= \case
+       Just (_, Crucible.AnyCFG cfg, _warns) -> k cfg
```

Comment:
There are quite a few call sites to `getTranslatedCFG`, such as this one, where the translation warnings are discarded. At the very least, it seems like we should call something like `handleWarning` below on the warnings. Even if `handleWarning` does nothing currently, it would at least provide a central place to fix the TODO should we ever want to do so.

Reply:
Good idea.  I don't think it's hard to implement something sensible, there just wasn't anything already.